CHANGELOG.md

```diff
@@ -12,4 +12,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Implement `XpringClient`'s `getBalance` method using rippled's protocol buffers. 
-- Implement `XpringClientDecorator`'s `getRawTransactionStatusMethod` using rippled's protocol buffers. 
```

Comment:
Why did this get removed?

Reply:
This didn't get removed. I re-wrote to be `XpringClient` to match the other descriptions. `XpringClientDecorator` isn't really publicly exposed so it doesn't make too much sense. 

I also changed  "`getRawTransactionStatusMethod`" to "`getRawTransactionStatus` method" which is just fixing a typo.